app/src/main/res/layout/item_custom_outcoming_text_message.xml

```diff
@@ -82,6 +84,20 @@
             app:layout_alignSelf="center"
             app:tint="@color/high_emphasis_text" />
 
+
+
+        <TextView
+            android:id="@+id/messageType"
```

Comment:
lets rename this id to something with "edit".
maybe
"@+id/messageEditIndicator"
?
same for item_custom_incoming_text_message.xml